src/OrbitGl/App.h

```diff
@@ -577,6 +577,12 @@ class OrbitApp final : public DataViewFactory,
   void ShowHistogram(const std::vector<uint64_t>* data, const std::string& scope_name,
                      uint64_t scope_id) override;
 
+  // Triggers symbol loading for all modules in ModuleManager that are not loaded yet. This is done
+  // with a simple prioritization. The module `ggpvlk.so` is queued to be loaded first, the "main
+  // module" (binary of the process) is queued to be loaded second. All other modules are queued in
+  // no particular order.
+  void LoadAllSymbols();
```

Comment:
IMO, the priorities for ggpvlk and the main module should be outside of this method. I would rather have a function to get prioritized modules and have this function who can receive a list of prioritized modules to load.

Reply:
done